src/Keboola/DbExtractor/Test/ExtractorTest.php

```diff
@@ -8,17 +8,13 @@
 
 class ExtractorTest extends \PHPUnit_Framework_TestCase
 {
-    /**
-     * @var string
-     */
+    public const CONFIG_FORMAT_YAML = 'yaml';
+    public const CONFIG_FORMAT_JSON = 'json';
+
+    /** @var string */
     protected $dataDir = ROOT_PATH . "/tests/data";
 
-    /**
-     * @param $driver
-     * @param string $format (yaml or json)
-     * @return array|mixed
-     */
-    protected function getConfig(string $driver, string $format = 'yaml'): array
+    protected function getConfig(string $driver, string $format = self::CONFIG_FORMAT_YAML): array
     {
         if ($format === 'json') {
```

Comment:
this could use the constant as well. Next branch is `=== CONFIG_FOMRAT_YAML` and only then there is "else" throw exception unknown format XY. Right now $format='ini' will break the code unexpectedly in Yaml parser.

Reply:
Could be switch + default throw exception if you feel like it

Reply:
Sorry I missed it earlier..